### DISCUSSION TITLE
Runtime Error: Sizes of tensors must match except in dimension 1. Expected size 46 but got size 45 for tensor number 1 in the list.

### DISCUSSION THREAD

**Discussion by easonoob** [basujindal/stable-diffusion#162] [Q&A]
Whenever I run the program, there is a runtime error that says `RuntimeError: Sizes of tensors must match except in dimension 1. Expected size 46 but got size 45 for tensor number 1 in the list.` I don't know if it is Stable Diffusion's source code's, or this optimized code's problem. Please help, thx.

**Comment by easonoob** [comment]
This is my input: `python optimizedSD/optimized_txt2img.py --prompt "A fantasy snowy mountain with a lake at front, surrounded by green trees, clear blue sky, magical, very photorealistic, real, unreal engine, ray tracing, faraway, gigantic, 8k, landscape" --H 960 --W 1440 --seed 27 --n_iter 2 --n_samples 1 --ddim_steps 50`

**Reply by easonoob** [reply]
When I set the width and height to 512, the error won't occur, is there a limitation of the size of the output image?

**Reply by easonoob** [reply]
Of course if it is too large, my 8GB CUDA memory will run out.

**Reply by easonoob** [reply]
My GPU is gtx 1080

**Reply by bitRAKE** [reply]
Both the hieght and width must be divisible by 64.
(1440 mod 64 != 0)

**Reply by easonoob** [reply]
Oh, ok, thanks!
